README.md

```diff
@@ -482,23 +486,123 @@ Combines variants across multiple batches, resolves complex variants, re-genotyp
 #### Outputs:
 * Finalized "cleaned" VCF and QC plots
 
-## <a name="module07">Module 07</a> (in development)
-Apply downstream filtering steps to the cleaned VCF to further control the false discovery rate; all steps are optional and users should decide based on the specific purpose of their projects.
+## <a name="join-raw-calls">JoinRawCalls</a>
 
-Filtering methods include:
-* minGQ - remove variants based on the genotype quality across populations.
-Note: Trio families are required to build the minGQ filtering model in this step. We provide tables pre-trained with the 1000 genomes samples at different FDR thresholds for projects that lack family structures, and they can be found at the paths below.  These tables assume that GQ has a scale of [0,999], so they will not work with newer VCFs where GQ has a scale of [0,99].
+Merges raw unfiltered calls across batches. Concordance between these genotypes and the joint call set usually can be indicative of variant quality and is used downstream for genotype filtering.
+
+#### Prerequisites:
+* [ClusterBatch](#cluster-batch)
+
+#### Inputs:
+* Clustered Manta, Wham, Scramble, Melt, and/or depth VCF URIs ([ClusterBatch](#cluster-batch))
+* Ped file
+* Reference sequence
+
+#### Outputs:
+* VCF of clustered raw calls
+* Ploidy table
+
+## <a name="svconcordance">SVConcordance</a>
+
+Computes genotype concordance metrics between all variants in the joint call set and raw calls.
+
+#### Prerequisites:
+* [MakeCohortVcf](#make-cohort-vcf)
+* [JoinRawCalls](#join-raw-calls)
+
+#### Inputs:
+* Cleaned ("eval") VCF URI ([MakeCohortVcf](#make-cohort-vcf))
+* Joined raw call ("truth") VCF URI ([JoinRawCalls](#join-raw-calls))
+* Reference dictionary URI
+
+#### Outputs:
+* VCF with concordance annotations
+
+## <a name="filter-genotypes">FilterGenotypes</a>
+
+Performs genotype quality recalibration using a machine learning model based on [xgboost](https://github.com/dmlc/xgboost), and filters genotypes. 
+
+The ML model uses the following features:
+
+* Genotype properties:
+  * Allele frequency (AF), no-call counts
```

Comment:
Copied these from one of Ted's messages and it occurs to me that this is confusing. The counts of non-ref alleles and no-calls are calculated for each genotype individually (so not the AF INFO field). Will keep on the same line since these are allele type counts.